server/sonar-ce/src/main/java/org/sonar/ce/app/CeServer.java

```diff
@@ -171,15 +171,18 @@ private void waitForStopSignal() {
         try {
           Thread.sleep(CHECK_FOR_STOP_DELAY);
         } catch (InterruptedException e) {
-          // ignore the interruption itself, check the flag
+          // ignore the interruption itself
+          // Do not propagate the isInterrupted flag with : Thread.currentThread().interrupt();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
False positive, in my opinion